### PR TITLE
Update RGBCube.py

### PR DESCRIPTION
**Pull request by Prabhnith**

### CODE REVIEW

File: examples.py/3D/Form/RGBCube.py [hunk 1]
```diff
@@ -1,21 +1,21 @@
...
- """
+'''
+ RGB Cube.
+ 
+ The three primary colors of the additive color model are Red, Green and Blue.
+ This RGB color cube displays a smooth transitions between these colors. 
+'''
  
 xmag, ymag = (0, 0)
 newXmag, newYmag = (0, 0)
```
Comment (GoToLoop) [inline examples.py/3D/Form/RGBCube.py]: `xmag = ymag = newXmag = newYmag = 0`  has a much clearer intent than `(0, 0)`.

File: examples.py/3D/Form/RGBCube.py [hunk 2]
```diff
@@ -27,14 +27,14 @@ def draw():
     diff = xmag - newXmag
```
Comment (GoToLoop) [inline examples.py/3D/Form/RGBCube.py]: `newXmag = TAU * mouseX / width`
`newYmag = TAU * mouseY / height`

File: examples.py/3D/Form/RGBCube.py [hunk 3]
```diff
@@ -1,21 +1,21 @@
-"""
- * RGB Cube.
- * 
- * The three primary colors of the additive color model are red, green, and blue.
- * This RGB color cube displays smooth transitions between these colors. 
- """
+'''
+ RGB Cube.
+ 
+ The three primary colors of the additive color model are Red, Green and Blue.
```
Comment (GoToLoop) [inline examples.py/3D/Form/RGBCube.py]: Dunno why the need to capitalize the 3 primary colors here?

File: examples.py/3D/Form/RGBCube.py [hunk 4]
```diff
@@ -1,21 +1,21 @@
...
+ This RGB color cube displays a smooth transitions between these colors.
```
Comment (GoToLoop) [inline examples.py/3D/Form/RGBCube.py]: We can't use the undefined article `a` for plural words like transitions!

**Comment by GoToLoop** [comment]
After checking the whole "RGB Cube" code, I believe making variables _newXmag_ & _newYmag_ global isn't right. B/c they're confined within **draw()** and they don't need retention of their previous values.
That is, they're initialized anew here: `newXmag, newYmag = TAU * mouseX / width, TAU * mouseY / height`

File: examples.py/3D/Form/RGBCube.py [hunk 5]
```diff
@@ -1,21 +1,21 @@
-"""
- * RGB Cube.
- * 
- * The three primary colors of the additive color model are red, green, and blue.
- * This RGB color cube displays smooth transitions between these colors. 
- """
+'''
```
Comment (jdf) [inline examples.py/3D/Form/RGBCube.py]: https://www.python.org/dev/peps/pep-0257/

> For consistency, always use ```"""triple double quotes"""``` around docstrings.

**Pull request closed without merging**